examples/gke-consumption-options/on-demand/README.md

```diff
@@ -0,0 +1,72 @@
+# On-demand GKE Consumption Option
+
+This example captures details on how the deployment yaml and blueprint should look like for a user interested in using on-demand capacity.
+
+Note that there is no reservation / spot / flex start variables included in the yamls.
+
+## Create a cluster
+These steps guide you through the cluster creation process.
+
+Note: If you create multiple clusters using these same cluster blueprints, ensure that all VPCs and subnet names are unique per project to prevent errors.
```

Comment:
Let's remove it from this PR given that its not a product ask and we have not tested this out.